addons/website_sale/tests/test_fuzzy.py

```diff
@@ -0,0 +1,68 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.fields import Command
+from odoo.tests import tagged
+from odoo.tests.common import TransactionCase
+
+
+@tagged('-at_install', 'post_install')
+class TestFuzzy(TransactionCase):
+    def test_variant_default_code(self):
+        website = self.env.ref('website.default_website')
+
+        product_attribute = self.env['product.attribute'].create({
+            'name': 'Color',
+            'display_type': 'color',
+            'value_ids': [
+                Command.create({
+                    'name': "Red",
+                    'sequence': 1,
+                }),
+                Command.create({
+                    'name': "Green",
+                    'sequence': 2,
+                }),
+                Command.create({
+                    'name': "Blue",
+                    'sequence': 3,
+                }),
+            ]
+        })
+        template = self.env['product.template'].create({
+            'name': 'A Colorful Product',
+            'attribute_line_ids': [
+                Command.create({
+                    'attribute_id': product_attribute.id,
+                    'value_ids': [Command.set(product_attribute.value_ids.ids)],
+                })
+            ]
+        })
```

Comment:
`ProductVariantsCommon` > `product_template_sofa` ?

please use the existing test commons when possible :pray: